Source/SpatialGDK/Private/SpatialActorChannel.cpp

```diff
@@ -541,20 +542,18 @@ void USpatialActorChannel::SetChannelActor(AActor* InActor)
 	// If the entity registry has no entry for this actor, this means we need to create it.
 	if (ActorEntityId == 0)
 	{
-		USpatialNetConnection* SpatialConnection = Cast<USpatialNetConnection>(Connection);
-		check(SpatialConnection);
-
-		// Mark this channel as being responsible for creating this entity once we have an entity ID.
-		bCreatingNewEntity = true;
-
-		// Reserve an entity ID for this channel.
-		TSharedPtr<worker::Connection> PinnedConnection = WorkerConnection.Pin();
-		if (PinnedConnection.IsValid())
+		// If the actor is stably named, we only want to start the creation process on one server (the one that
+		// has the Global State Manager) to avoid having multiple copies of replicated stably named actors in SpatialOS
+		if (InActor->IsFullNameStableForNetworking())
 		{
-			ReserveEntityIdRequestId = PinnedConnection->SendReserveEntityIdRequest(0);
+			/*USpatialPackageMapClient* PackageMap = Cast<USpatialPackageMapClient>(SpatialNetDriver->GetSpatialOSNetConnection()->PackageMap);
```

Comment:
Delete commented code

Reply:
Ah, yes, sorry - commited a mildly WIP version of the branch, should have told you to hold off on reviewing. There is a bug where the PackageMap desyncs with the current implementation, so I will have to look into it thoroughly.

Reply:
Basically SpatialPackageMapClient (FSpatialNetGUIDCache) has two maps: NetGUIDToUnrealObjectRef and UnrealObjectRefToNetGUID and they should always hold the same data, just in reverse order. But with the current implementation, they are *wildly* inconsistent, and it is driving me crazy, since there is only one point of entry into these maps and that point of entry (FSpatialNetGUIDCache::RegisterObjectRef) ensures that they are indeed consistent.